src/gui/src/ServerConfigDialogBase.ui

```diff
@@ -17,7 +17,7 @@
    <item>
     <widget class="QTabWidget" name="m_pTabWidget">
      <property name="currentIndex">
-      <number>2</number>
+      <number>0</number>
      </property>
```

Comment:
Why was this changed?

Reply:
The dialog was opening on the incorrectly opening with Advanced tab and not the screens one.

Reply:
Ah, okay. Can you split this change into a new PR, please? That way we can isolate different changes.